website/docs/r/netapp_volume_quota_rule.html.markdown

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "NetApp"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_netapp_volume_quota_rule"
+description: |-
+  Manages a Volume Quota Rule.
+---
+
+# azurerm_netapp_volume_quota_rule
+
+Manages a Volume Quota Rule.
+
+## Example Usage
+
+```hcl
+resource "azurerm_netapp_volume_quota_rule" "example" {
+  name                = "example"
+  resource_group_name = "example"
+  location            = "West Europe"
+  quota_size_in_kib   = 42
+  account_name        = "example"
+  pool_name           = "example"
+  volume_name         = "example"
+  quota_type          = "IndividualUserQuota"
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `account_name` - (Required) The name of the NetApp Account where the volume is located. Changing this forces a new Volume Quota Rule to be created.
+
+* `location` - (Required) The Azure Region where the Volume Quota Rule should exist. Changing this forces a new Volume Quota Rule to be created.
+
+* `name` - (Required) The name which should be used for this Volume Quota Rule. Changing this forces a new Volume Quota Rule to be created.
+
+* `pool_name` - (Required) The name of the NetApp pool in which the NetApp Volume belongs to. Changing this forces a new Volume Quota Rule to be created.
+
+* `quota_size_in_kib` - (Required) Quota size in kibibytes.
```

Comment:
I don't believe this is required

Reply:
This is quite the oppposite, it was required, then I experimented with what we have discussed around mib and gib making all optional but when we decided to rollback I forgot to move it back to required, good catch.